app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    check = command.split(" ")
+    if len(check) == 3:
+        if check[0] != "cp" or check[1] == check[2]:
```

Comment:
The condition `check[1] == check[2]` checks if the source and destination files are the same. While this prevents overwriting the source file, it might be beneficial to provide feedback to the user or log this event instead of silently returning `None`.